src/components/LineCollection/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { createYScale, createXScale } from '../../utils/scale-helpers';
+import ScalerContext from '../../context/Scaler';
+import Line from '../Line';
+
+export const LineCollection = props => {
+  const { series, width, height, domain } = props;
+  const xScale = createXScale(domain, width);
```

Comment:
It seems domain is used instead of baseDomain, is there any purpose for doing that?

Reply:
The `LineCollection` and other rendering components do not know about baseDomain/subdomain. All it does is take in a domain and render it with the data.

Reply:
Oh, just found that in props, so domain is basically a subDomain here. It turns out using React Context makes it a bit hard to keep track on all of the props available.